appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: '{build}'
+
+image: Visual Studio 2017
+
+# Test against the latest version of this Node.js version
+# environment:
+#   nodejs_version: "6"
+
+platform:
+  - x64
+
+cache:
```

Comment:
At this point, despite extensive research, I don’t have a definitive answer, but I’ve included this with the understanding that it enables us to target the x64 architecture, as we intend.  According to the AppVeyor documentation (https://www.appveyor.com/docs/appveyor-yml/), this sets the platform for MSBuild, with options of `x86`, `x64` and `Any CPU`.  After cloning the repo, “AppVeyor runs MSBuild to build project sources and package artifacts.”  (https://www.appveyor.com/docs/build-phase/)  A number of the examples I’ve found use the `x64` setting, though without explanation.  See, e.g., electron (https://github.com/electron/electron/blob/master/appveyor.yml) (also includes `x86`), electron-rebuild (https://github.com/electron/electron-rebuild/blob/master/.appveyor.yml) (an open-source contribution from Slack), spawn-rx (https://github.com/paulcbetts/spawn-rx/blob/master/appveyor.yml) (also a Slack contribution), and electron-forge (https://github.com/electron-userland/electron-forge/blob/master/.appveyor.yml) (from a member of the Slack team).  Atom also uses this config (together with `x86`) (https://github.com/atom/atom/blob/master/appveyor.yml).  

BTW, my research also suggests that AppVeyor has three “build worker images”, Visual Studio 2013, Visual Studio 2015 and Visual Studio 2017.  The OS for the first two is Windows Server 2012 R2, and for the third, Windows Server 2016.  See https://www.appveyor.com/docs/build-environment/.  As I understand, Windows 7's server counterpart is Windows Server 2008 R2, and thus it may be that AppVeyor does not build for the Windows 7 platform.